enterprise/security/pom.xml

```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>org.apache.shiro</groupId>
       <artifactId>shiro-core</artifactId>
-      <version>1.3.2</version>
+      <version>1.4.0</version>
```

Comment:
Don't do this. look at how versions are centrally managed for all other dependencies